docs/conf.py

```diff
@@ -36,7 +36,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["myst_parser", "sphinxcontrib.programoutput"]
+extensions = ["myst_parser", "sphinxcontrib.programoutput", "sphinxcontrib.apidoc"]
```

Comment:
Let's keep them sorted
```suggestion
extensions = [
    "myst_parser",
    "sphinxcontrib.apidoc",
    "sphinxcontrib.programoutput",
]
```